apps/survey/test/survey.js

```diff
@@ -1,347 +1,557 @@
-const { assertInvalidOpcode, assertRevert } = require('@aragon/test-helpers/assertThrow')
-const getBlockNumber = require('@aragon/test-helpers/blockNumber')(web3)
-const timeTravel = require('@aragon/test-helpers/timeTravel')(web3)
-
-const getContract = name => artifacts.require(name)
-const pct16 = x => new web3.BigNumber(x).times(new web3.BigNumber(10).toPower(16))
-const createdSurveyId = receipt => receipt.logs.filter(x => x.event == 'StartSurvey')[0].args.surveyId
-
-contract('Survey app', accounts => {
-  let daoFact, surveyBase, survey
-
-  let ABSTAIN_VOTE, ANY_ENTITY
-  let APP_MANAGER_ROLE, CREATE_SURVEYS_ROLE, MODIFY_PARTICIPATION_ROLE
-
-  const surveyTime = 1000
-  const root = accounts[0]
-  const holder19 = accounts[0]
-  const holder31 = accounts[1]
-  const holder50 = accounts[2]
-  const nonHolder = accounts[4]
-  const NULL_ADDRESS = '0x00'
-  const minimumAcceptanceParticipationPct = pct16(20)
+const {
+  assertInvalidOpcode,
+  assertRevert
+} = require("@aragon/test-helpers/assertThrow");
+const getBlockNumber = require("@aragon/test-helpers/blockNumber")(web3);
+const timeTravel = require("@aragon/test-helpers/timeTravel")(web3);
+
+const getContract = name => artifacts.require(name);
+const pct16 = x =>
+  new web3.BigNumber(x).times(new web3.BigNumber(10).toPower(16));
+const createdSurveyId = receipt =>
+  receipt.logs.filter(x => x.event == "StartSurvey")[0].args.surveyId;
+
+contract("Survey app", accounts => {
+  let daoFact, surveyBase, survey;
+
+  let ABSTAIN_VOTE, ANY_ENTITY;
+  let APP_MANAGER_ROLE, CREATE_SURVEYS_ROLE, MODIFY_PARTICIPATION_ROLE;
+
+  const surveyTime = 1000;
+  const root = accounts[0];
+  const holder19 = accounts[0];
+  const holder31 = accounts[1];
+  const holder50 = accounts[2];
+  const nonHolder = accounts[4];
+  const NULL_ADDRESS = "0x00";
+  const minimumAcceptanceParticipationPct = pct16(20);
```

Comment:
Why are those all these changes? I don't have a strong opinion about format issues, but we don't usually use trailing semicolon for JS.

Reply:
Ah crap that must be my linter, I'll revert them.